packages/common/client-utils/src/bufferNode.ts

```diff
@@ -46,9 +46,10 @@ export type IsoBuffer = Buffer;
  *
  * @internal
  */
-export function Uint8ArrayToString(arr: Uint8Array, encoding?: string): string {
-	// Make this check because Buffer.from(arr) will always do a buffer copy
-	return Buffer.isBuffer(arr) ? arr.toString(encoding) : Buffer.from(arr).toString(encoding);
+export function Uint8ArrayToString(arr: Uint8Array, encoding?: "utf8" | "base64"): string {
+	// Buffer extends Uint8Array.  Therefore, 'arr' may already be a Buffer, in
+	// which case we can avoid copying the Uint8Array into a new Buffer instance.
+	return (Buffer.isBuffer(arr) ? arr : Buffer.from(arr)).toString(encoding);
```

Comment:
TS5 complained about 'arr.toString(...)' because it failed to see that 'Buffer.isBuffer(...)' had narrowed the type of 'arr' to 'Buffer'.  Tweaking the expressing this way fixed the issue.